packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -115,6 +115,12 @@ export type CallCompositeOptions = {
    */
   devicePermissions?: DevicePermissionRestrictions;
   /* @conditional-compile-remove(call-readiness) */
+  /**
+   * Opt in call readiness feature for your call
+   * setting this to True will add call readiness feature in call experience
+   */
+  callReadinessOptedIn?: boolean;
```

Comment:
I prefer we do not use the term "optedIn" for when Contoso would like to use Call Readiness features.
Possibly rename to something like:
"enableCallReadiness"
or 
"enableCallReadinessChecks"
etc..

Reply:
@JamesBurnside should we name it optedin or enable? When we were syncing up on this we named it callReadinessOptedIn

Reply:
I think we should write a one-pager on this for the team to review. The doc should cover naming, as well as discussing if there is a time from which we should enable by default instead of opt-in (e.g. "after a year if we do a major bump we should look to convert all opt-ins to byDefault features"), and if possible contrast this API with Calling's current feature API.
I agree the suffix `OptedIn` isn't quite right, but I think getting the team's opinion is best here as it will need to scale to more features

Reply:
We are going to keep it in its current state then iterate on API name